dts/xtensa/intel/intel_adsp_ace15_mtpm.dtsi

```diff
@@ -163,6 +163,11 @@
 			compatible = "intel,adsp-shim";
 			reg = <0x71f00 0x100>;
 		};
+		
+		dfpmccap: dfpmccap@71b00 {
+			compatible = "intel,adsp-dfpmccap";
+			reg = <0x71b00 0x4000000>;
```

Comment:
I dont think it is really 0x4000000

Reply:
Updated:
https://github.com/zephyrproject-rtos/zephyr/pull/52205/files#diff-84b915da18690fe4c2bab81fc9a29ec0c9919ea9fdc71b265a9079e6d95eedb0